api/src/policies/notifications-policy.ts

```diff
@@ -12,6 +12,8 @@ export class NotificationsPolicy extends PolicyFactory(Notification) {
   }
 
   create(): boolean {
+    if (this.record.userId === this.user.id) return true
```

Comment:
I think I would still block create/destroy since these should be created/destroyed by the system, not the user. I'd only permit "update" since that implies the ability to mark as read/unread.